packages/contracts/contracts/L1/MockL2OutputOracle.sol

```diff
@@ -55,7 +56,7 @@ contract MockL2OutputOracle {
      * checkpointed.
      */
     function nextTimestamp() public view returns (uint256) {
```

Comment:
What do you think about having an internal `_nextTimestamp` that this wraps and then call the internal function in `appendL2Output`?

Reply:
I tested this out, and it appears to make no zero difference, which is nice actually.